doi-requests-commons/src/main/java/no/unit/nva/doi/requests/userdetails/UserDetails.java

```diff
@@ -11,6 +12,11 @@
     public static final JsonPointer APPLICATION_ROLES = JsonPointer.compile(
         "/authorizer/claims/custom:applicationRoles");
 
+    @JacocoGenerated
+    private UserDetails() {
```

Comment:
private constructor for class with only static methods because PMD and Jacoco complain